tools/ci_build/github/azure-pipelines/stages/jobs/react-natvie-andriod-e2e-test-job.yml

```diff
@@ -44,9 +44,8 @@ jobs:
       versionSpec: '20.x'
 
   - script: |
-      sudo apt install coreutils ninja-build nodejs npm yarn
-      npm install --global yarn
-    displayName: Install coreutils, ninja, npm, and yarn
+      sudo apt install coreutils ninja-build nodejs npm
```

Comment:
if we already used `NodeTool` above, why do we need apt install nodejs and npm?

Reply:
Only thing I did to this line is remove the `yarn`. Everything elese stays the same.

Reply:
We can remove it, but also want to have consistent dependcie across dev environment. I would suggest make a separate PR to lockdown nodejs in package.json.